recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "black" %}
-{% set version = "21.5b1" %}
+{% set version = "21.5b2" %}
 
 package:
   name: {{ name|lower }}-recipe
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 23695358dbcb3deafe7f0a3ad89feee5999a46be5fec21f4f1d108be0bcdb3b1
+  sha256: a85cfc9d2295c0ae7172ad8b49823abb11627709ba1bd661ffb00f184da85280
```

Comment:
```suggestion
  sha256: 1fc0e0a2c8ae7d269dfcf0c60a89afa299664f3e811395d40b1922dff8f854b5
```

Reply:
Yes, I see, thanks - I should have got the SHA from the PyPI file instead of the GitHub file!